release_notes/ocp-4-5-release-notes.adoc

```diff
@@ -149,6 +148,19 @@ You can now use CSI to specify volumes directly in the Pod specification, rather
 
 Volume cloning using CSI, previously in Technology Preview, is now fully supported in OpenShift Container Platform 4.5. For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-cloning.adoc#persistent-storage-csi-cloning[CSI volume cloning].
 
+[id="ocp-4-5-olm"]
+=== Operator Lifecycle Manager
+
+[id="ocp-4-5-olm-v1-crd"]
+==== v1 CRD support
+
+Operator Lifecycle Manager (OLM) now supports Operators using v1 Custom Resource
+Definitions (CRDs) in catalogs and on cluster. Previously, OLM only supported
+v1beta1 CRDs when loading Operators into catalogs and deploying them on cluster.
```

Comment:
I would add something like "OLM now manages both v1 and v1beta1 CRDs in the same way" so that users don't think that we dropped support for `v1beta1` CRDs in 4.5. As a sidenote, kube 1.19 is dropping support for v1beta1 CRDs and that will potentially cause a lot of issues for our community operators. In 4.6 we will no longer be able to support v1beta1 CRDs.